resources/public/api/conf/1.0/amend_employment_expenses.raml

```diff
@@ -0,0 +1,38 @@
+is:
+  - headers.acceptJson
+  - headers.contentTypeJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.formatTaxYear
+  - errors.formatValue
+  - errors.incorrectOrEmptyBody
+  - errors.ruleTaxYearNotSupported
+  - errors.ruleTaxYearRangeInvalid
+  - errors.ruleTaxYearNotEnded
```

Comment:
tax year not ended error documentation scenario doesn't match the spec, it also mentions "brought forward" losses